### ISSUE TITLE
Cache name textbox not limited to one line

### ISSUE THREAD

**Issue by culmor30** [culmor30/cgeo-wear#7]
I intended for the textbox to scroll horizontally when displaying long cache names rather than expand upward, so that the compass would stay centered on the screen. Apparently that isn't working.

**Comment by Lineflyer** [comment]
Indeed: I can confirm this on my LG G Watch R when navigating to the lab caches on the recent Mainz GIGA event. Those cache names have been extraordinary long.
